prov/efa/src/rxr/rxr.h

```diff
@@ -676,11 +678,14 @@ static inline void efa_eq_write_error(struct util_ep *ep, ssize_t err,
 	FI_WARN(&rxr_prov, FI_LOG_EQ,
```

Comment:
Since we already print err information earlier, we just print `Unable to write to EQ` here.